### ISSUE TITLE
Didn't find pmd violation while has pmd.xml

### ISSUE THREAD

**Issue by biaomingzhong** [jenkinsci/violation-comments-to-github-plugin#15]
My pmd.xml :
```
<?xml version="1.0" encoding="UTF-8"?>
<pmd version="5.5.2" timestamp="2017-12-06T11:55:38.278">
<file name="/Users/quinlanzhong/Projects/test-Meijiabang/meijiaLove/src/main/java/com/meijialove/fragments/community/CommunityFragment.java">
<violation beginline="28" endline="120" begincolumn="8" endcolumn="1" rule="ClassMustHaveAuthorRule" ruleset="AlibabaJavaComments" package="com.meijialove.fragments.community" class="CommunityFragment" priority="3">
【CommunityFragment】缺少包含@author的注释信息
</violation>
<violation beginline="35" endline="35" begincolumn="27" endcolumn="32" rule="LowerCamelCaseVariableNamingRule" ruleset="AlibabaJavaNaming" package="com.meijialove.fragments.community" class="CommunityFragment" method="newInstance" variable="Refram" priority="3">
变量名【Refram】不符合lowerCamelCase命名风格
</violation>
<violation beginline="115" endline="116" begincolumn="9" endcolumn="19" rule="NeedBraceRule" ruleset="AlibabaJavaFlowControl" package="com.meijialove.fragments.community" class="CommunityFragment" method="topClick" priority="3">
if语句缺少大括号
</violation>
</file>
</pmd>
```

while pmd.xml is at `build/reports/quality/pmd.xml`. My regex is `.*/reports/quality/pmd.*\.xml`. But still no violation found, It contains warning I need to comment at PR

**Comment by tomasbjerre** [comment]
I think it is because `externalInfoUrl``attribute is missing in the report file.

That is a bug and is fixed here:
https://github.com/tomasbjerre/violations-lib/issues/27

I'm working on releasing the fix. I'm migrating from Central to Bintray and as soon as that is done, this fix will be released here.

**Comment by biaomingzhong** [comment]
Is there any work around solution for me ?

**Comment by tomasbjerre** [comment]
Not that I know of.

**Comment by tomasbjerre** [comment]
Should be fixed in 1.49. Available in Jenkins update center within a few hours.

**Comment by tomasbjerre** [comment]
Open issue again if not working.
